src/views/orb-view.ts

```diff
@@ -230,6 +232,13 @@ export class OrbView<N extends INodeBase, E extends IEdgeBase> implements IOrbVi
         this._settings.interaction.isDragEnabled =
           settings.interaction.isDragEnabled;
       }
+
+      // Check if isZoomEnabled is a boolean value
```

Comment:
Btw you can remove these comments (including the ones for the drag check). It is usually good practice to just have a comment when you need to describe a complex part of the code. The code here is already well written so it doesn't need comments.